bot/kodiak/cli.py

```diff
@@ -93,3 +94,42 @@ def refresh_pull_requests() -> None:
     from kodiak.refresh_pull_requests import main
 
     main()
+
+
+@cli.command()
+def http_server() -> None:
+    """
+    run HTTP server used for receiving webhooks
+    """
+    from kodiak.main import main
+
+    main()
+
+
+@cli.command()
+def queue_consumers() -> None:
+    """
+    run the consumers for the various queues
+    """
+    from kodiak.queue import main
+
+    main()
+
+
+@cli.command()
+def http_server_and_consumers() -> None:
+    """
+    Run http server and consumers as subprocesses for backwards compat with self
+    hosting docs.
+    """
+    from kodiak.main import main as http_main
+    from kodiak.queue import main as queue_consumers_main
+
+    http_process = Process(target=http_main)
+    http_process.start()
+
+    queue_consumers_process = Process(target=queue_consumers_main)
+    queue_consumers_process.start()
+
+    http_process.join()
+    queue_consumers_process.join()
```

Comment:
If queue_consumers_process dies before http_process, what happens? Don't we want to exit immediately in that case?

Reply:
sounds good